src/components/CookieBanner/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { CookieConsent } from 'react-cookie-consent';
+import { useTranslation } from 'react-i18next';
+
+import usePlugins from '../../hooks/usePlugins';
+import { isCookieConsentIntegration } from '../../plugin';
+
+import './index.less';
+
+export const CookieBanner = () => {
+  const plugins = usePlugins();
+
+  const { t } = useTranslation();
+
+  const getPlugin = () => {
+    let cookiePlugin: React.ReactElement | null = null;
+
+    // TODO: warn if more than one cookie-plugin has been detected
+
+    plugins.forEach(plugin => {
+      if (isCookieConsentIntegration(plugin.integration)) {
+        const {
+          key, wrappedComponent: WrappedPluginComponent
+        } = plugin;
+
+        cookiePlugin = <WrappedPluginComponent key={key} />;
+      }
+    });
+
+    return cookiePlugin;
+  };
+
+  return (
+    <div className="CookieBanner">
+      <CookieConsent
+        key="default-cookie-consent"
+        location="bottom"
+        buttonText={t('CookieBanner.button')}
+        cookieName="SHOGun-Session-Cookie"
```

Comment:
Is this name used somewhere? If so, the backend doesn't make use of any session cookies and it should be adapted.

Reply:
As far as I see it, it's not used anywhere. It is intended that the user agrees to this if they continue to use the application. I don't think the backend needs to be modified to do this, otherwise the user will have no choice but to leave the page.  Otherwise we should discuss it further.

Reply:
The cookie mentioned here is used exclusively in the client and is saved in the browser storage within the cookies area when accepted by the user so that this message is no longer displayed when the client is called up again (until the cookie has been removed or has expired). If the user does not agree, this message will prompt again each time the client is opened.

Reply:
Thanks for the clarification!